### PR TITLE
chore(macros/EmbedLiveSample): deprecate screenshot URL parameter

### PR DESCRIPTION
**Pull request by bsmth**
## Summary

The EmbedLiveSample macro doesn't use the `$3` param (screenshot URL) in EN content.
Adding a `deprecatedParams` flaw for it

## Screenshots

Screenshot 2024-03-13 at 11 55 49 AM

## How did you test this change?

* yarn && yarn dev
* visit http://localhost:3000
* Added the updated macro to a document and visited on `http://localhost:3000/...#_flaws`

__Related issues and pull requests:__

Removals PRs:

- [x] https://github.com/mdn/content/pull/32588
- [x] https://github.com/mdn/content/pull/32505

### CODE REVIEW

File: kumascript/macros/EmbedLiveSample.ejs [hunk 1]
```diff
@@ -15,12 +15,19 @@
...
-// We have deprecated the use of the $4 to $6 parameters
-// If one of this parameter is not empty, we raise a MacroDeprecationError flaw
-// When there will be 0 use left on translated-content (May 2022: 203 occurrences for $4),
+// We have deprecated the $3, $4, and $5 parameters.
+// If a non-empty value is provided for a deprecated parameter, we raise a MacroDeprecationError flaw
+// When there are 0 uses in translated-content,
 // we will be able to simplify this macro (and likely the whole of MDN).
-if ($4 !== "" || $5 !== "") {
-  mdn.deprecatedParams("The fourth and fifth parameters of 'EmbedLiveSample' are deprecated");
+// (May 2022: 203 occurrences for $4)
```
Comment (caugner) [inline kumascript/macros/EmbedLiveSample.ejs:22]: Should we remove this line, as it's probably outdated?
```suggestion
```

**Review by caugner — APPROVED** [review]
LGTM, just one suggestion.

File: kumascript/macros/EmbedLiveSample.ejs (hunk 1 above)
Comment (bsmth) [reply kumascript/macros/EmbedLiveSample.ejs:22]: I'm fine with that, let's get rid of it

**Comment by bsmth** [comment]
Thank you!

**Pull request merged**